pkg/gofr/datasource/redis/redis.go

```diff
@@ -102,6 +110,8 @@ func getRedisConfig(c config.Config) *Config {
 		options.Password = redisConfig.Password
 	}
 
+	options.DB = redisConfig.DB
```

Comment:
Can we log the db index to which the redis DB is connected ? in connection successful logs ?

Reply:
done